espanol/capitan/trueno.md

```diff
@@ -1 +1,5 @@
-El Capitán Trueno estaba volviendo en ese momento de su campaña en las tierras de Kirlian
+El Capitán Trueno estaba volviendo en ese momento de su campaña en las tierras de Kirlian.
+
+En su camino de regreso se encontro con una bruja. 
+
+[Una bruja ofrece ayuda](bruja/bruja.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.